pull-request/release-please/action.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Manage Release
+description: |
+  This action manages auto-release handling for the repository.
+  Based on the configuration in the release-please-config.json file in conjunction with
+  all conventional commits since the last release, it will create a new release PR.
+  If the PR itself gets merged, the action will create a new release and tag it.
+  The config-filename is required until https://github.com/googleapis/release-please-action/issues/1017 is resolved.
```

Comment:
I don't understand the logic here - if there is no other ways to get the file instead of reading it from the current repo, why we need to require the filename? opting for a default config filename in all the repositories would make less parameters to pass.